src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -180,10 +180,10 @@
   "minecolonies.config.citizencallforhelp.comment": "Should citizens call Guards for help when attacked?",
   "minecolonies.config.rangerenchants": "Archer Enchants",
   "minecolonies.config.rangerenchants.comment": "Should Archer Guards benefit from Power/Punch/Flame enchants (on bows)?",
-  "minecolonies.config.rangerdamagemult": "Archer Damage Multiplier",
-  "minecolonies.config.rangerdamagemult.comment": "Damage multiplier for Archer Guards.",
-  "minecolonies.config.knightdamagemult": "Knight Damage Multiplier",
-  "minecolonies.config.knightdamagemult.comment": "Damage multiplier for Knight Guards.",
+  "minecolonies.config.rangerdamagemult": "Archer Attack Multiplier",
+  "minecolonies.config.rangerdamagemult.comment": "Attack multiplier for Archer Guards.",
+  "minecolonies.config.knightdamagemult": "Knight Attack Multiplier",
+  "minecolonies.config.knightdamagemult.comment": "Attack multiplier for Knight Guards.",
```

Comment:
I'm not happy with "Attack" here, very generic, damage is more clear. you can call it Attack Damage too, but only attack is not enough